### ISSUE TITLE
pino v7 upgrade

### ISSUE THREAD

**Issue by rethab** [probot/pino#171]
hey 👋 , [pino v7 has been released](https://github.com/pinojs/pino/releases/tag/v7.0.0). are there any plans to upgrade this library to use that new version? if you don't have time, would you be open to a PR?

**Comment by gr2m** [comment]
Yes, we would love to update. I looked into it but it was much more involved and I couldn't figure it out in the available time. 

Any help with the upgrade would be greatly appreciated

**Comment by jetersen** [comment]
This is currently blocking as this is not ESM ready.

**Comment by jetersen** [comment]
I would like to see if I cannot tackle this one 😅

**Comment by gr2m** [comment]
Awesome, thanks!

**Comment by jetersen** [comment]
Only thing left is the tests and I am somewhat stuck on them. Not sure how to properly adapt them.

**Comment by dgellow** [comment]
Any plan to upgrade to [v8](https://github.com/pinojs/pino/releases/tag/v8.0.0)?

**Comment by gr2m** [comment]
pull requests welcome. We are in the process o updating probot/probot to its latest dependencies, which will also drop support for Node < 18. We should be fine with upgrading pino all the way to v8, but we need to go to v7 first, make tests pass, then upgrade to v8
